things/index.html

```diff
@@ -23,6 +24,7 @@
         alt="Denise Wright Designs Logo"
         class="logo"
       />
+      <!-- when your nav shrinks the dropdown menu pops up and disappears, consider modifying the animation to prevent this -->
```

Comment:
I can remove this animation completely to eliminate that strange pop up when the browser shrinks, however, It would make the drawer opening very fast and rough looking. I chose to keep it the same because it is very unlikely that a user would even notice that seeing as that is a tablet and mobile view that wouldn't be seen upon loading on a mobile device. 
So unless my users just start shrinking their browser for the fun of it, it should be fine.